docs/typescript.md

```diff
@@ -5,92 +5,43 @@ title: Using TypeScript
 
 import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
 
-[TypeScript][ts] is a language which extends JavaScript by adding type definitions, much like [Flow][flow]. While React Native is built in Flow, it supports both TypeScript _and_ Flow by default.
+[TypeScript][ts] is a language which extends JavaScript by adding type definitions. New React Native projects target TypeScript by default, but also support JavaScript and Flow.
 
 ## Getting Started with TypeScript
 
-If you're starting a new project, there are a few different ways to get started.
+New projects created by the [React Native CLI](http://localhost:3000/docs/next/environment-setup#creating-a-new-application) or popular templates like [Ignite][ignite] will use TypeScript by default.
 
-You can use the [TypeScript template][ts-template]:
-
-```shell
-npx react-native init MyApp --template react-native-template-typescript
-```
-
-:::note
-
-If the above command is failing, you may have an old version of `react-native` or `react-native-cli` installed globally on your system. To fix the issue try uninstalling the CLI:
-
-- `npm uninstall -g react-native-cli` or `yarn global remove react-native-cli`
-
-and then run the `npx` command again.
-Optionally, you can also use the command given below to get started with your template.
-
-- `npx react-native --ignore-existing init MyApp --template react-native-template-typescript`
-
-:::
-
-You can use [Expo][expo], which maintains TypeScript templates, or will prompt you to automatically install and configure TypeScript when a `.ts` or `.tsx` file is added to your project:
-
-<Tabs groupId="package-manager" defaultValue={constants.defaultPackageManager} values={constants.packageManagers}>
-<TabItem value="npm">
+TypeScript may also be used with [Expo][expo], which maintains TypeScript templates, or will prompt you to automatically install and configure TypeScript when a `.ts` or `.tsx` file is added to your project.
 
 ```shell
 npx create-expo-app --template
```

Comment:
one question about this -- why do we need to pass the empty `--template` flag? 

By the cli docs it seems to imply it'll guide you through to choose the template but trying out it doesn't seem to?

```
❯ npm create expo-app --template
✔ What is your app named? … my-app
✔ Downloaded and extracted project files.
✔ Installed JavaScript dependencies.

✅ Your project is ready!

To run your project, navigate to the directory and run one of the following npm commands.

- cd my-app
- npm run android
- npm run ios
- npm run web

~ via  v18.7.0 took 36s
❯ cd my-app/
```

Reply:
Without the `--template` the default template is used without listing the options.

Reply:
Ah, I see the problem. Running `npm create expo-app` seems to do not forward the flag, that's probably why we recommend `npx create-expo-app` over it on this page in the package Readme.

Reply:
with `npm create` you need to use `npm create expo-app -- --template` in order to differentiate between flags for `npm create` and flags for `create-expo-app`

Reply:
Ohoo my mistake! I think I had copied the wrong command over from some other documentation, thanks for clarifying!